CONTRIBUTING.md

```diff
@@ -116,11 +115,7 @@ git push --set-upstream origin release-1.7
 
 Once the commit has been cherry picked, the author will need to open a PR merging to the release branch, release-1.7 for example.
 
-## Engage with us
```

Comment:
This section already exists on L17.

https://github.com/kyverno/kyverno/blob/a37901425fbab76fb79a777a961f0af613a69a2d/CONTRIBUTING.md?plain=1#L17-L21